app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
avoid nesting and use https://developer.android.com/reference/androidx/fragment/app/FragmentContainerView